src/main/java/service/WriterService.java

```diff
@@ -0,0 +1,6 @@
+package service;
+
+public interface WriterService {
+    void writeToFile(String outputPath);
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```